desc/coils.py

```diff
@@ -1153,9 +1153,15 @@ def from_symmetry(cls, coils, NFP=1, sym=False):
 
         """
         if not isinstance(coils, CoilSet):
-            coils = CoilSet(coils)
-
-        [_check_type(coil, coils[0]) for coil in coils]
+            try:
+                coils = CoilSet(coils)
+            except (TypeError, ValueError):
```

Comment:
the code cov here is outdated, these lines are covered if you check the code cov report